lib/resources/example_json/dstu2_examples_slot.rb

```diff
@@ -40,709 +40,719 @@ module Resources
+                  },
+                  "extension": [
+                      {
+                          "url": "https://fhir-ehr.cerner.com/dstu2/StructureDefinition/scheduling-location",
+                          "valueReference": {
+                              "reference": "Location/4048303"
+                          }
+                      }
+                  ],
+                  "type": {
+                      "coding": [
+                          {
+                              "system": "http://snomed.info/sct",
+                              "code": "394581000",
+                              "display": "Community medicine",
+                              "userSelected": false
+                          },
+                          {
+                              "system": "https://fhir.cerner.com/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/codeSet/14249",
+                              "code": "4734719",
+                              "display": "Lab Visit",
+                              "userSelected": true
+                          }
+                      ],
+                      "text": "Lab Visit"
+                  },
+                  "schedule": {
+                      "reference": "Schedule/4734719-4048303-7045347-40"
+                  },
+                  "freeBusyType": "free",
+                  "start": "2019-01-01T14:40:00.000Z",
+                  "end": "2019-01-01T14:55:00.000Z"
+              },
+              "search": {
+                  "mode": "match"
               }
-            ],
-            "type": {
-              "coding": [
-                {
-                  "system": "http://snomed.info/sct",
-                  "code": "394581000",
-                  "display": "Community medicine",
-                  "userSelected": false
-                },
-                {
-                  "system": "https://fhir.cerner.com/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/codeSet/14249",
-                  "code": "21265426",
-                  "display": "Same Day",
-                  "userSelected": true
-                }
-              ],
-              "text": "Same Day"
-            },
-            "schedule": {
-              "reference": "Schedule/21265426-4048128-3121781-370"
-            },
-            "freeBusyType": "free",
-            "start": "2016-01-30T14:10:00.000-06:00",
-            "end": "2016-01-30T15:10:00.000-06:00"
           }
-        }
       ]
-    }
+  }
 
     DSTU2_SLOT_BUNDLE_INCLUDE_SCHEDULE ||= {
       "resourceType": "Bundle",
-      "id": "b1d02f24-a47e-44db-a608-82f75fef8fec",
+      "id": "7934637e-9b43-4e57-b919-b2cae00f0a43",
       "type": "searchset",
       "link": [
-        {
-          "relation": "self",
-          "url": "https://fhir-open.sandboxcerner.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Slot?slot-type=http%3A%2F%2Fsnomed.info%2Fsct%7C394581000&schedule.actor=Practitioner%2F2578010&start=2016&_include=Slot%3Aschedule&_count=5"
```

Comment:
This can't be helped if the response body has been updated with the removal of the param. The response body `url` field will display all valid params as they're entered in the request URL. Once the param is added back, per my request [above](https://github.com/cerner/fhir.cerner.com/pull/364#discussion_r404916776), this should be resolved.

Reply:
see comment above